lighthouse-core/scripts/cleanup-LHR-for-diff.js

```diff
@@ -23,9 +23,21 @@ writeFileSync(filename, cleanAndFormatLHR(data), 'utf8');
  * @return {string}
  */
 function cleanAndFormatLHR(lhrString) {
+  /** @type {LH.Result} */
   const lhr = JSON.parse(lhrString);
-  delete lhr.configSettings.auditMode;
```

Comment:
was there an issue with having this previously? Was it only because we were generating it more manually?

Reply:
its just this:

```js
auditMode": "./lighthouse-core/test/results/artifacts",
```

hopefully it isn't an absolute path in some cases?